actions/issues.go

```diff
@@ -149,3 +227,42 @@ func requestParamToQueryFilter(query, paramValue, paramName *string) {
 		}
 	}
 }
+
+func preCacheIssues(whereClause string, params buffalo.ParamValues, page string) {
+	tx, err := models.DB.NewTransaction()
+	defer tx.Close()
+
+	if err != nil {
+		fmt.Println(errors.New("no transaction found"))
+	}
+
+	cacheConn := cache.CachePool.Get()
+	defer cacheConn.Close()
+
+	issues := &models.Issues{}
+	nextPage, _ := strconv.Atoi(page)
+	nextPage++
+	nextPageStr := strconv.Itoa(nextPage)
+	nextParams := params.(url.Values)
+	nextParams.Set("page", nextPageStr)
+	nextQ := tx.PaginateFromParams(nextParams).Eager()
+
+	nextCacheKey := "issues:" + whereClause + "and page=" + nextPageStr
+
+	ok, _ := cache.Exists(&cacheConn, nextCacheKey)
+
+	if ok == false {
```

Comment:
```suggestion
	if !ok {
```

Reply:
actually I did this what if ok  = `nil`. is !nil allowed?

Reply:
oops ok is never nil